bool dir_l
uint16 speed_l
bool dir_r
uint16 speed_r
